### DISCUSSION TITLE
Can I limit access to environment variables?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [vercel/community#3856] [Feature Question] [answered]
### Question

Hi there.

I have a user with the role "Member".

To my surprise, he is capable of viewing environment variables.

Is it possible to restrict users from viewing environment variables?

**Comment by amyegan** [comment, marked as answer]
Hey @alexbjorlig. There are currently three [access roles](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#) available on Pro teams.

- [Owner Role](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#owner-role) - full access
- [Member Role](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#member-role) - cannot edit team settings, invite new users to team, change roles of other users on team, or manage billing info
- [Billing Role](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#billing-role) - can only manage billing info

I'm not sure about restricting access to environment variables specifically. I'll share this with the team that manages that feature.
